libs/application/templates/parental-leave/src/lib/messages.ts

```diff
@@ -226,12 +261,27 @@ export const m = defineMessages({
     description:
       'Do you want to transfer up to one month of your parental leave rights to the other parent?',
   },
+  giveRightsDaysTitle: {
+    id: 'pl.application:request.rights.daysTitle',
+    defaultMessage: 'Hve marga daga viltu gefa?',
+    description: 'How many days would you like to give?',
+  },
   giveRightsDescription: {
     id: 'pl.application:give.rights.description',
     defaultMessage:
-      'Þú getur yfirfært allt að einn mánuð af þínum réttindum yfir á hitt foreldrið',
+      'Þú getur yfirfært allt að 45 dagar af þínum réttindum yfir á hitt foreldrið',
     description: 'You can give the other parent one month of your rights',
```

Comment:
Need change here as well, something like that
```suggestion
    description: 'You can give up to 45 days of your rights to the other parent ',
```

Reply:
Good catch, fixed now.